packages/cozy-harvest-lib/src/components/AccountModal.spec.jsx

```diff
@@ -99,10 +99,6 @@ describe('AccountModal', () => {
 
       component.find('AccountSelectBox').prop('onCreate')()
       expect(mockHistoryPush).toHaveBeenCalledWith('/new')
-
-      const accountTabs = component.find('KonnectorAccountTabs')
-      accountTabs.prop('addAccount')()
-      expect(mockHistoryPush).toHaveBeenCalledWith('/new')
```

Comment:
ok but should we add it in another test?

Reply:
My understanding of this test was wrong. Now the test is kept and I removed this commit.